README.md

```diff
@@ -285,6 +285,20 @@ By default, the style-loader appends `<style>` elements to the end of the style
 }
 ```
 
+A new `<style>` element can be inserted before a specific element by passing an object with `type='before'`, e.g.
+
+**webpack.config.js**
+```js
+{
+  loader: 'style-loader'
+  options: {
+    insertAt: {
+        before: '#style-overrides'

```

Comment:
The options table  needs an `type` update aswell `{String} => {String\|Object}`  (⚠️ `\` [escaped])